pkg/kubelet/kubelet_node_status.go

```diff
@@ -216,16 +218,47 @@ func (kl *Kubelet) reconcileCMADAnnotationWithExistingNode(node, existingNode *v
 	return true
 }
 
+// EnumerateDistribution resolves the underlying distribution
+func (kl *Kubelet) EnumerateDistribution() (string, string) {
```

Comment:
This doesn't use the receiver. can just be a pure function without being on `Kubelet`.

Also, please name your return values if you're returning (string, string)